src/configuration-store/hybrid.store.ts

```diff
@@ -7,6 +8,9 @@ export class HybridConfigurationStore<T> implements IConfigurationStore<T> {
     protected readonly servingStore: ISyncStore<T>,
     protected readonly persistentStore: IAsyncStore<T> | null,
   ) {}
+  private environment: Environment;
+  private configFetchedAt: string;
+  private configPublishedAt: string;
```

Comment:
🐛 These should be nullable for the case configuration is not fetched yet + these fields are not set in the constructor. (Same in all other stores.)

I also wonder why typescript didn't catch that. It looks like we don't have `strict` mode enabled. (This case specifically is covered by `strictPropertyInitialization`.)

Reply:
Great find!